paddle/operators/lookup_table_op.cc

```diff
@@ -51,9 +54,12 @@ class LookupTableOpMaker : public framework::OpProtoAndCheckerMaker {
     AddInput("W",
              "An input represents embedding tensors,"
              " which is a learnable parameter.");
-    AddInput("Ids",
-             "An input with type int32 or int64"
-             "contains the ids to be looked up in W.");
+    AddInput(
+        "Ids",
+        "An input with type int32 or int64"
+        "contains the ids to be looked up in W."
+        "Ids must be a column vector with rank = 2."
+        "The 1st dimension is batch size, and the 2nd dimension must be 1");
```

Comment:
Remove: “The 1st dimension is batch size”,  since for sequence index input, the 1st dimension is the total of words number in this mini-batch, not batch size.

Reply:
Done